arbeitszeit_flask/database/repositories.py

```diff
@@ -1676,7 +1688,9 @@ def perform(self) -> int:
             .values(**self.changes)
             .execution_options(synchronize_session="fetch")
         )
-        return self.db.session.execute(sql_statement).rowcount  # type: ignore
```

Comment:
Do we no longer ignore the type?  Is that the type of rowcount?

Reply:
I do not know exactly why we originally instructed mypy to ignore the type check here. Anyway, I noticed that this instruction is not necessary anymore, because mypy does not complain anymore. Probably because we improved the typing in this module in the last few commits.